spyder/widgets/tests/test_Upper_Lower.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
```

Comment:
Please add the header found in all other python files (that include the copyright)

Please change the name of this file. On Python we have lowercase names. Also we need to add what widget we are testing so

`spyder/widgets/tests/test_Upper_Lower.py` -> `spyder/widgets/tests/test_codeeditor_upper_lower.py`